src/main/java/uk/gov/pay/api/service/ConnectorService.java

```diff
@@ -34,4 +42,18 @@ public Charge getCharge(Account account, String paymentId) {
 
         throw new GetChargeException(response);
     }
+
+    public RefundsResponse getPaymentRefunds(Account account, String paymentId) {
+        Response connectorResponse = client
+                .target(connectorUriGenerator.refundsForPaymentURI(account.getAccountId(), paymentId))
+                .request()
+                .get();
+
+        if (connectorResponse.getStatus() == SC_OK) {
+            RefundsFromConnector refundsFromConnector = connectorResponse.readEntity(RefundsFromConnector.class);
+            return RefundsResponse.valueOf(refundsFromConnector, baseUrl);
```

Comment:
I don't think `ConnectorService` should know about `publicApiConfig`.
It should return `RefundsFromConnector` and map it in `GetPaymentRefundsService` which is higher lever class.

Reply:
Agree! It will move with further refactoring